docs/GettingStartedDocs/Contributors/building_oe_sdk.md

```diff
@@ -63,7 +64,18 @@ make install
 
 Open Enclave samples can be found in ~/openenclave/share/openenclave/samples
 
-See [Open Enclave samples](/samples/README.md) for details.
+See [Open Enclave samples](/samples/README_Linux.md) for details.
+
+### On Windows
+Assuming you install the SDK as below (also described in the [basic install section](WindowsInstallInfo.md#basic-install-on-windows))
+
+```bash
+cmake .. -G  Ninja -DNUGET_PACKAGE_PATH= [NuGet_package_Path]   -DCMAKE_INSTALL_PREFIX:PATH=C:\openenclave" -DUSE_LIBSGX=ON
+ninja install
+```
+Open Enclave samples can be found in c:\openenclave\share\openenclave\samples
+
+See [Open Enclave samples](/samples/README_Windows.md) for details.
 
 ## Using the Open Enclave SDK
```

Comment:
The section after this probably needs to be split since they link to using_oe_sdk.md, which is only for the Linux side now.